lib/matplotlib/artist.py

```diff
@@ -1114,13 +1114,44 @@ def matchfunc(x):
 
     def get_cursor_data(self, event):
         """
-        Get the cursor data for a given event.
+        Return the cursor data for a given event.
+
+        .. note::
+            This is part of the internal API. As an end-user of Matplotlib
+            you will most likely not call this method yourself.
```

Comment:
I don't particularly like the wording here, there are in fact a lot of other "public" parts of the Matplotlib API that are "intended as internal/private" (even more than this API)...  I guess I'd prefer just something like "intended to be overridden by artist subclasses" two paragraphs below.